coralproject/talk
# Make Approve/Reject button styles more robust for longer labels

## What does this PR do?

For other languages than English the 'Approve' text did not fit into the button. (We are currently working on a German localization of Talk.)

This commit makes the CSS a bit more flexible by removing some very pixel-specfic styles. (Also, imho, I think that aligning the buttons to the right side of the comment box increases usability when scrolling through a list of comments when moderating.)

## How do I test this PR?

Check the Approved/Rejected lanes in the admin interface. Buttons are aligned a bit more to the right. If you change the buttons' text to something longer than 'Approved' (e.g. the German 'Freigegeben') the button does not break anymore by having a ugly newline in it.

Before:
![bildschirmfoto am 2018-01-30 um 14 35 36](https://user-images.githubusercontent.com/32779/35570027-36efaf80-05ce-11e8-9080-805c335ffb9b.png)

After:
![bildschirmfoto am 2018-01-30 um 14 36 18](https://user-images.githubusercontent.com/32779/35570029-373392d6-05ce-11e8-921c-2347ef190758.png)
